src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java

```diff
@@ -0,0 +1,21 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OuterTypeNumber"/>
+  </module>
+</module>
+*/
+
+// xdoc section -- start
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;// violation, Outer types defined is '3' (max allowed is '1')
+
+public class Example1 {
+}
+
+class Example {
+    void exampleMethod() {}
+}
+
+enum Outertype {
```

Comment:
OuterType and please remove from whitelist file changes. 
We try to keep whitelist in good state

Reply:
done